### DISCUSSION TITLE
Create repo and upload file

### DISCUSSION THREAD

**Discussion by LYF123123** [community/community#5809] [Mobile]
create repo, create new pr and upload file on ipad

**Comment by kubeczki** [comment]
This would be super great now that vscode.dev is available

**Comment by grinsonofdgun0147** [comment]
Coin market cap solution

**Comment by brianlovin** [comment]
This would be awesome, agreed. Thanks for the suggestion @LYF123123!

**Comment by benhatsor** [comment]
A related project is [codeit.codes](https://codeit.codes), from which you can create and edit GitHub repos on mobile.

**Comment by aeveltstra** [comment]
Yes, please. And create folders, too.

**Comment by HenSquared** [comment]
You can now create repositories on iOS and Android :) 
https://github.blog/changelog/2026-05-11-create-repositories-on-the-go-with-github-mobile/
